http/src/response/status_code.rs

```diff
@@ -95,6 +113,12 @@ mod tests {
         Sync
     );
 
+    #[test]
+    fn test_eq_with_integer() {
+        assert_eq!(200u16, StatusCode::new(200));
+        assert_eq!(404u16, StatusCode::new(404));
```

Comment:
shouldn't this be flipped to ensure it compiles when on either side?

Reply:
Sure. Done.